bloco_4/dia_3/ex02.js

```diff
@@ -0,0 +1,19 @@
+// 2-   Para o segundo exercício, faça o mesmo que antes, mas
+//      que imprima um triângulo retângulo com 5 asteriscos de base.
+//      Por exemplo:
+
+//      n = 5
+
+//      *
+//      **
+//      ***
+//      ****
+//      *****
+
+let n = 3;
+let lineTriangle = '';
+
+for(let line = 0; line < n; line += 1){
+  lineTriangle += ' *';
```

Comment:
Acredito que deveria tirar esse espaço para ficar igual ao pedido no exercício, o que acha?
```suggestion
  lineTriangle += '*';
```

Reply:
Show Lucas! Tinha colocado esse espaço só pra ficar mais nítido as alterações no console. Acabei esquecendo de retirar kkk. Muito obrigado! :rocket: